src/HistoryService.Mappers/Responses/FindServiceHistoryResponseMapper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using LT.DigitalOffice.HistoryService.Mappers.Responses.Interfaces;
+using LT.DigitalOffice.HistoryService.Models.Db;
+using LT.DigitalOffice.HistoryService.Models.Dto.Responses;
+
+namespace LT.DigitalOffice.HistoryService.Mappers.Responses
+{
+  public class FindServiceHistoryResponseMapper : IFindServiceHistoryResponseMapper
+  {
+    public ServiceHistoryInfo Map(DbServiceHistory dbServiceHistory)
+    {
+      if (dbServiceHistory == null)
+      {
+        return null;
+      }
+
+      return new ServiceHistoryInfo
+      {
+        Id = dbServiceHistory.Id,
+        ServiceId = dbServiceHistory.ServiceId,
```

Comment:
insert service data, returning the identifier is not informative for the user

Reply:
done